.travis.yml

```diff
@@ -45,4 +45,4 @@ before_script:
 script:
   - make docker-bootstrap
   - make test
-  - make test-ssas
+  #- make test-ssas
```

Comment:
Why commented out?

Reply:
Isn't this supposed to run in the `bcda-ssas-app` repo moving forward whenever Travis gets configured?

Reply:
Yup!   I was just wondering why it was commented out instead of removed.

Reply:
Oh! I commented out all SSAS related code throughout the repo rather than remove it since I wasn't sure when the cut over would be or what the conclusion about new SSAS code was( will code continue in both BCDA and SSAS repos or just SSAS repo)